evalml/tests/component_tests/test_components.py

```diff
@@ -888,6 +888,10 @@ def test_transformer_transform_output_type(X_y_binary):
                 component, SelectByType
             ):
                 assert transform_output.shape == (X.shape[0], 0)
+            elif isinstance(component, RFRegressorSelectFromModel):
+                assert transform_output.shape == (X.shape[0], 2)
+            elif isinstance(component, RFClassifierSelectFromModel):
+                assert transform_output.shape == (X.shape[0], 5)
```

Comment:
What's the `2` or `5` values coming from here?

Reply:
it's the number of columns selected by the FS component using the default parameters on `X_y_binary` and `X_y_regression`.